docs/Doxyfile

```diff
@@ -1734,7 +2018,7 @@ EXTRA_SEARCH_MAPPINGS  =
 # Configuration options related to the LaTeX output
 #---------------------------------------------------------------------------
 
-# If the GENERATE_LATEX tag is set to YES, doxygen will generate LaTeX output.
+# If the GENERATE_LATEX tag is set to YES, Doxygen will generate LaTeX output.
 # The default value is: YES.
 
 GENERATE_LATEX         = YES
```

Comment:
We should disable this since we aren't using it. IMO having it on broadcasts and intent that someone is consuming this output and verifying that it works well.